docs/topics/cli-run.adoc

```diff
@@ -25,56 +27,66 @@ $ ./mta-cli --input /path/to/jee-example-app-1.0.0.ear --output /path/to/output
+== {ProductShortName} command examples
 
 [discrete]
 === Running {ProductShortName} on an application archive
 
 The following command analyzes the `com.acme` and `org.apache` packages of the link:https://github.com/windup/windup/blob/master/test-files/jee-example-app-1.0.0.ear[jee-example-app-1.0.0.ear] example EAR archive for migrating from JBoss EAP 5 to JBoss EAP 7:
 
+[source,terminal]
 ----
-$ <MTA_HOME>/bin/mta-cli --input /path/to/jee-example-app-1.0.0.ear --output /path/to/report-output/ --source eap:5 --target eap:7 --packages com.acme org.apache
+$ <MTA_HOME>/bin/mta-cli \
+    --input /path/to/jee-example-app-1.0.0.ear \
+    --output /path/to/report-output/ --source eap:5 --target eap:7 \
+    --packages com.acme org.apache
 ----
 
 [discrete]
 === Running {ProductShortName} on source code
 
 The following command analyzes the `org.jboss.seam` packages of the link:https://github.com/windup/windup/tree/master/test-files/seam-booking-5.2[seam-booking-5.2] example source code for migrating to JBoss EAP 6.
 
-[source,options="nowrap",subs="+quotes"]
+[source,terminal]
 ----
-$ <MTA_HOME>/bin/mta-cli --sourceMode --input /path/to/seam-booking-5.2/ --output /path/to/report-output/ --target eap:6 --packages org.jboss.seam
+$ <MTA_HOME>/bin/mta-cli --sourceMode --input /path/to/seam-booking-5.2/ \
+    --output /path/to/report-output/ --target eap:6 --packages org.jboss.seam
 ----
 
 [discrete]
 === Running cloud-readiness rules
 
 The following command analyzes the `com.acme` and `org.apache` packages of the link:https://github.com/windup/windup/blob/master/test-files/jee-example-app-1.0.0.ear[jee-example-app-1.0.0.ear] example EAR archive for migrating to JBoss EAP 7. It also evaluates for cloud readiness:
 
-[source,options="nowrap",subs="+quotes"]
+[source,terminal]
 ----
-$ <MTA_HOME>/bin/mta-cli --input /path/to/jee-example-app-1.0.0.ear --output /path/to/report-output/ --target eap:7 --target cloud-readiness --packages com.acme org.apache
+$ <MTA_HOME>/bin/mta-cli --input /path/to/jee-example-app-1.0.0.ear \
+    --output /path/to/report-output/ \
+    --target eap:7 --target cloud-readiness --packages com.acme org.apache
 ----
 
 [discrete]
 === Overriding {ProductShortName} properties
 
 To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler` argument on the command line. For example, to use the _Procyon_ decompiler, use the following syntax:
 
-[source,options="nowrap",subs="+quotes"]
+[source,terminal]
 ----
-$ <MTA_HOME>/bin/mta-cli -Dwindup.decompiler=procyon --input <INPUT_ARCHIVE_OR_DIRECTORY> --output <OUTPUT_REPORT_DIRECTORY> --target <TARGET_TECHNOLOGY> --packages <PACKAGE_1> <PACKAGE_2>
+$ <MTA_HOME>/bin/mta-cli -Dwindup.decompiler=procyon \
+    --input <INPUT_ARCHIVE_OR_DIRECTORY> --output <OUTPUT_REPORT_DIRECTORY> \
+    --target <TARGET_TECHNOLOGY> --packages <PACKAGE_1> <PACKAGE_2>
 ----
 
-[discrete]
+// == Using OpenRewite recipes
+include::using-openrewrite-recipes.adoc[leveloffset=+1]
```

Comment:
Modules are included in assemblies, not other modules. Please move the OR module include to the master.adoc assembly.